src/dash-playlist-loader.js

```diff
@@ -178,7 +316,48 @@ export default class DashPlaylistLoader extends EventTarget {
       this.trigger('mediachanging');
     }
 
-    // TODO: check for sidx here
+    if (playlist.sidx) {
```

Comment:
perhaps it would make more sense to more the mediaRequest below this if statement into `if (!playlist.sidx)` and then return inside of it? That would keep similar code together and get rid of a big if scope.

Reply:
Makes sense to me